airbyte-integrations/connectors/destination-mssql-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLWriterTest.kt

```diff
@@ -0,0 +1,156 @@
+                            rawId =
+                                rs.getString(COLUMN_NAME_AB_RAW_ID)?.let { UUID.fromString(it) },
+                            extractedAt =
+                                Instant.ofEpochMilli(rs.getLong(COLUMN_NAME_AB_EXTRACTED_AT)),
+                            loadedAt = null,
+                            generationId = rs.getLong(COLUMN_NAME_AB_GENERATION_ID),
+                            data = objectValue,
+                            airbyteMeta =
+                                rs.getString(COLUMN_NAME_AB_META)?.let {
+                                    val meta =
+                                        Jsons.deserialize(it, AirbyteRecordMessageMeta::class.java)
+                                    OutputRecord.Meta(
+                                        changes =
+                                            meta.changes
+                                                .map { change ->
+                                                    Meta.Change(
+                                                        field = change.field,
+                                                        change = change.change,
+                                                        reason = change.reason,
+                                                    )
+                                                }
+                                                .toList(),
+                                        syncId =
+                                            meta.additionalProperties["syncId"]
+                                                ?.toString()
+                                                ?.toLong()
+                                    )
+                                },
+                        )
+                    output.add(record)
+                }
+            }
+        }
+        return output
+    }
+
+    override fun dumpFile(
+        spec: ConfigurationSpecification,
+        stream: DestinationStream
+    ): List<String> {
+        return emptyList()
+    }
+
+    private fun getConfiguration(
+        spec: ConfigurationSpecification,
+        stream: DestinationStream
+    ): MSSQLConfiguration {
+        /*
+         * Replace the host, port and schema to match what is exposed
+         * by the container and generated by the test suite in the case of the schema name
+         */
+        val configOverrides =
+            mutableMapOf("host" to MSSQLContainerHelper.getHost()).apply {
+                MSSQLContainerHelper.getPort()?.let { port -> put("port", port.toString()) }
+                stream.descriptor.namespace?.let { schema -> put("schema", schema) }
+            }
+        return MSSQLConfigurationFactory()
+            .makeWithOverrides(spec = spec as MSSQLSpecification, overrides = configOverrides)
+    }
+}
+
+class MSSQLDataCleaner : DestinationCleaner {
+    override fun cleanup() {
+        // TODO("Not yet implemented")
```

Comment:
checking: we don't actually need this class, b/c we're doing everything in testcontainers? (i.e. we could swap to the noop cleaner)